service/service_test.go

```diff
@@ -556,12 +556,16 @@ func assertMetrics(t *testing.T, metricsAddr string, expectedLabels map[string]l
 		"otelcol_process_runtime_heap_alloc_bytes":       false,
 		"otelcol_process_runtime_total_alloc_bytes":      false,
 		"otelcol_process_uptime":                         false,
+		"promhttp_metric_handler_errors_total":           false,
```

Comment:
@codeboten, is this metric expected to be added? I'm not sure if it provides any value to the collector internal metrics...

Reply:
it's added by the prometheus server that is started via the SDK. we might be able to drop it via a view

Reply:
ok, that sounds good. Let me create an issue